services/app-api/src/domain-models/index.ts

```diff
@@ -45,7 +45,9 @@ export type {
     RateFormDataType,
     PackageStatusType,
     ContractPackageSubmissionType,
+    RatePackageSubmissionType,
     ContractPackageSubmissionWithCauseType,
+    RatePackageSubmissionWithCauseType,
```

Comment:
Should RatePackageSubmissionWithCauseType be added to codegen like we have for ContractPackageSubmissionWithCauseType?

Reply:
I see it in the codegen ✅